Commandant/ArgumentParser.swift

```diff
@@ -125,7 +125,11 @@ public final class ArgumentParser {
 		rawArguments.removeAll()
 
 		var foundValue: String?
-		argumentLoop: for var index = 0; index < oldArguments.count; index++ {
+		var index = 0
+
+		while index < oldArguments.count {
+			defer { index++ }
```

Comment:
We should avoid `++` too: https://github.com/apple/swift-evolution/blob/master/proposals/0004-remove-pre-post-inc-decrement.md